cmd/jaeger-plugin-proxy/main.go

```diff
@@ -0,0 +1,96 @@
+// This file and its contents are licensed under the Apache License 2.0.
+// Please see the included NOTICE for copyright information and
+// LICENSE for a copy of the license.
+
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"os"
+
+	"gopkg.in/yaml.v2"
+
+	"github.com/hashicorp/go-hclog"
+	"github.com/hashicorp/go-plugin"
+	"github.com/jaegertracing/jaeger/plugin/storage/grpc/shared"
+	"github.com/jaegertracing/jaeger/proto-gen/storage_v1"
+	"github.com/timescale/promscale/pkg/jaegerproxy"
+	"google.golang.org/grpc"
+)
+
+func main() {
+	logger := hclog.New(&hclog.LoggerOptions{
+		Level:      hclog.Warn,
+		Name:       "jaeger-plugin-proxy",
+		JSONFormat: true,
+	})
+
+	var configAddr string
+	flag.StringVar(&configAddr, "config", "", "Configuration file address")
+	flag.Parse()
+
+	bSlice, err := ioutil.ReadFile(configAddr)
+	if err != nil {
+		logger.Error("Invalid configuration file address, exiting.")
+		os.Exit(1)
+	}
+
+	conf := new(jaegerproxy.ProxyConfig)
+	if err = yaml.Unmarshal(bSlice, conf); err != nil {
+		logger.Error("Unmarshalling configuration file contents", "err", err.Error())
+		os.Exit(1)
+	}
+
+	//todo: Remove later
+	logger.Warn("configuration", "conf", fmt.Sprintf("%v", conf))
```

Comment:
Nit: I guess `Info` would be more appropriate to print configuration. I did notice a comment above but I think logging configuration is useful when running/debugging in prod.

Reply:
Yes, Info is useful but Jaeger only allows Warn & Error level logs to be printed out.